gruntfile.js

```diff
@@ -208,7 +208,7 @@ module.exports = function (grunt) {
   require('load-grunt-tasks')(grunt);
 
   // Making grunt default to force in order not to break the project.
```

Comment:
about the comment yeah, but we have this:
`grunt.option('force', false);`
if we leave it there in the gruntfile then even if you do `grunt test --force` the force option won't override it and it will force the build to fail in any case. If we do remove this option then grunt indeed defaults to failing tests but you can override it with the --force option from cmdline.

WDYT we should choose?